src/multi-level-checkbox-editor/node-box.tsx

```diff
@@ -61,7 +63,12 @@ const countSelectedChildren = (node: Node) => {
 const isNodeChecked = (node: Node) => {
     return hasChildren(node) && countSelectedChildren(node) === node.children!.length;
 };
-
+const getInfo = (info: string) => {
```

Comment:
The logic inside this function does not seem to make much sense. Returns empty string in any case

Reply:
removed it